polymer.externs.js

```diff
@@ -722,6 +722,9 @@ Polymer.dom = function(nodeOrEvent) {};
 
 Polymer.dom.flush = function() {};
 
+/** @param {!function()} debouncer */
```

Comment:
I don't think this param type is correct. See https://github.com/Polymer/polymer/blob/master/src/lib/debounce.html#L33.